content/en/docs/Responses/ArtistID3.md

```diff
@@ -9,13 +11,20 @@ description: >
 {{< tab header="**Example**:" disabled=true />}}
 {{< tab header="OpenSubsonic" lang="json">}}
 {
-  "id": "37ec820ca7193e17040c98f7da7c4b51",
-  "name": "2 Mello",
-  "coverArt": "ar-37ec820ca7193e17040c98f7da7c4b51_0",
-  "albumCount": 1,
-  "userRating": 5,
-  "artistImageUrl": "https://demo.org/image.jpg",
-  "starred": "2017-04-11T10:42:50.842Z"
+    "id": "37ec820ca7193e17040c98f7da7c4b51",
+    "name": "2 Mello",
+    "coverArt": "ar-37ec820ca7193e17040c98f7da7c4b51_0",
+    "albumCount": 1,
+    "userRating": 5,
+    "artistImageUrl": "https://demo.org/image.jpg",
+    "starred": "2017-04-11T10:42:50.842Z",
+    "musicBrainzId": "189002e7-3285-4e2e-92a3-7f6c30d407a2",
+    "sortName": "Mello (2)",
+    "roles": [
```

Comment:
Just thought: do we assume we skip subroles here?

Reply:
Yes as said we don't want to force support on servers, so there probably won't be an API to filter on subroles.
Clients will be able to filter on artist have role "performer" then if they want to filter on subrole will filter at client side.